app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -3,59 +3,65 @@
 namespace App\Http\Services\Tabbar;
 
 use Illuminate\Http\Request;
-use App\Http\Consts\UrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\HomeUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\ContentsUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\DocsUrlInfo;
+use App\Http\Consts\UrlInfo\Tabbar\Navigation\DevUrlInfo;
 
 /**
- * TabbarService
+ * TabbarService.
  *
  * タブバーの加工を行う
- *
  */
-class TabbarService {
-
+class TabbarService
+{
     /**
      * Create a new docs controller instance.
-     *
-     * @return void
      */
-    public function __construct() {
-        
+    public function __construct()
+    {
     }
 
     /**
-     * タブの情報を取得する
+     * タブの情報を取得する.
      * 
      * @param Request リクエストパラメータ
+     *
      * @return array 各tabで表示するためのURL
-     * 
      */
-    public function getTabbarInfo($request) {
+    public function getTabbarInfo($request)
+    {
         // リクエストパラメータからタブの情報を取得する
         $tabbar = $request->only([
             'home',
             'contents',
             'docs',
-            'dev'
+            'dev',
         ]);
-        // UrlInfoのインスタンス取得
-        $urlInfo = new UrlInfo();
+
+        $homeUrlInfo = new HomeUrlInfo();
+        $contentsUrlInfo = new ContentsUrlInfo();
+        $docsUrlInfo = new DocsUrlInfo();
+        $devUrlInfo = new DevUrlInfo();
+
         return [
-            'HOME' => $this->_getHomeTabInfo($tabbar['home'], $urlInfo->getHomeTabUrlInfo()),
-            'CONTENTS' => $this->_getContentsTabInfo($tabbar['contents'], $urlInfo->getContentsTabUrlInfo()),
-            'DOCS' => $this->_getDocsTabInfo($tabbar['docs'], $urlInfo->getDocsTabUrlInfo()),
-            'DEV' => $this->_getDevTabInfo($tabbar['dev'], $urlInfo->getDevTabUrlInfo())
+            'HOME' => $this->_getHomeTabInfo($tabbar['home'], $homeUrlInfo->getHomeTabUrlInfo()),
+            'CONTENTS' => $this->_getContentsTabInfo($tabbar['contents'], $contentsUrlInfo->getContentsTabUrlInfo()),
+            'DOCS' => $this->_getDocsTabInfo($tabbar['docs'], $docsUrlInfo->getDocsTabUrlInfo()),
+            'DEV' => $this->_getDevTabInfo($tabbar['dev'], $devUrlInfo->getDevTabUrlInfo()),
         ];
     }
 
     /**
-     * Homeタブの情報を取得する
+     * Homeタブの情報を取得する.
      * 
-     * @param string $homeParam リクエストパラメータのhomeの値
-     * @param array $homeTabUrlInfo Homeタブ内に表示するページのURL一覧
+     * @param string $homeParam      リクエストパラメータのhomeの値
+     * @param array  $homeTabUrlInfo Homeタブ内に表示するページのURL一覧
+     *
      * @return string Homeタブに表示するページのURL
-     * 
      */
-    private function _getHomeTabInfo($homeParam, $homeTabUrlInfo) {
+    private function _getHomeTabInfo($homeParam, $homeTabUrlInfo)
```

Comment:
__[code_sniffer]__
* Method name "_getHomeTabInfo" should not be prefixed with an underscore to indicate visibility